hotham/src/asset_importer/mod.rs

```diff
@@ -176,6 +207,21 @@ fn load_models_from_gltf_data(import_context: &mut ImportContext) -> Result<()>
     Ok(())
 }
 
+fn get_collider_mesh_ids(nodes: gltf::iter::Nodes) -> Vec<usize> {
+    let mut mesh_ids = Vec::new();
+    for node in nodes {
+        if !node.name().unwrap_or_default().contains(COLLIDER_TAG) {
+            continue;
+        }
+
+        if let Some(mesh) = node.mesh() {
+            mesh_ids.push(mesh.index())
+        }
+    }
+
+    mesh_ids
```

Comment:
Did you consider using `filter_map` for this?

Reply:
I did, but it started to look quite messy. 😦